src.ts/transaction.ts

```diff
@@ -1,3 +1,6 @@
+'use strict';
+
+import { BN } from "bn.js";
```

Comment:
I think we shouldn't use `bn.js`, but use https://github.com/GoogleChromeLabs/jsbi which is forward-compatible with native JS BigInt.

Reply:
added comment explaining why we use it.